quilt/tools/command.py

```diff
@@ -306,10 +306,44 @@ def push(package):
     pkgobj = PackageStore.find_package(owner, pkg)
     if pkgobj is None:
         raise CommandException("Package {owner}/{pkg} not found.".format(owner=owner, pkg=pkg))
+
+    obj_hashes = sorted(set(find_object_hashes(pkgobj.get_contents())))
+    total = len(obj_hashes)
+
+    headers = {
+        'Content-Encoding': 'gzip'
+    }
+
+    for idx, obj_hash in enumerate(obj_hashes):
+        print("Uploading %s (%d/%d)..." % (obj_hash, idx + 1, total))
```

Comment:
oh do we lose filename feedback to the user here? ideally we'd have that. because people won't recognize hashes. if non-trivial then backlog that feature.